PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkClusterListRefreshableCombo.kt

```diff
@@ -227,3 +230,13 @@ class CosmosServerlessSparkAccountsCombo: SparkClusterListRefreshableCombo() {
 
     override val viewModel: SparkClusterListRefreshableCombo.ViewModel by lazy { ViewModel() }
 }
+
+class ArcadiaSparkClusterListRefreshableCombo: SparkClusterListRefreshableCombo() {
```

Comment:
We need to override `comboBoxName` with `sparkComputeListComboBox` for distinguish different kind of spark cluster telemetries.